.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,28 @@
-## Summary
 
-*Briefly describe the issue and please DELETE the non applicable headings below. *
+### Observed behavior
 
-## System information
+description of the behavior that was observed, including screenshots or other references when applicable
 
-*Please specify the Kolibri version you were using and your operating system!*
+### Expected behavior
 
- - Version: ?
- - Operating system: ?
- - Browser: ?
+description of what behavior was expected but did not occur
 
-## Traceback or relevant snippet from server.log or browser console
+### User-facing consequences
 
-```
-INSERT TRACEBACK, LOG MESSAGES ETC. HERE
-```
+implications for a user of the application
```

Comment:
Who is "a user"? :)

Reply:
learners, coaches, admins, curators, etc